draft-ietf-rats-eat.md

```diff
@@ -2029,7 +2029,7 @@ This early allocation will presumably complete correctly
 * Claim Name: SW Manifests
 * Claim Description: Manifests describing the SW installed on the entity
 * JWT Claim Name: "manifests"
-* Claim Key: TBD273
+* Claim Key: TBD
```

Comment:
The CDDL defintion for this Claim includes "manifests-type = [+ $$manifest-formats]", which means that you can express multiple CoSWIDs or SUIT Manifests as content of an EAT. As CoSWIDs are either "Evidence" (which literally translates to Claims produced by an Attester) or "Payload" ("which means they are Endorsements that can be co-located with endorsed software components on an Attester), why should they be produced by the Verifier and included in Attestation Results? See below, the output of an appraisal conducted by a Verifier that is "swresult".